ts/features/it-wallet/saga/itwCredentialsSaga.ts

```diff
@@ -54,3 +61,12 @@ export function* handleCredentialsAddPid(
     );
   }
 }
+
+/**
+ * This saga handles the required checks before adding a credential.
+ * @param action the request dispatched action with a credential as payload.
+ * action: ActionType<typeof itwCredentialsChecks.request>
+ */
+export function* handleCredentialsChecks(): SagaIterator {
+  yield* put(itwCredentialsChecks.success());
+}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion(make decisions explicit):**  Besides the name and the description of the function, this handler does no‏ checks at all. This is totally fine, as we are just preparing the saga for the check to be implemented in subsequent iterations.

I'd just like to advise on having this decision **explicit** for the reader. I suggest three approaches:
1. place a big comment (i.e. "// TODO: skipping checks for now because bla bla bla... ")
2. name the function itself `handleSkipCredentialsChecks` or  `handleNoCredentialsChecks` (this makes the decision explicit in the saga, too)
3. don't scaffold checks until we make them (this is good to avoid premature optimization / premature abstraction)

Reply:
I guess the comment slipped away in the code as the intent was exactly to introduce a placeholder where we know for sure the check will be implemented.

Reply:
I'd like to provide some context regarding the decision not to scaffold the "checks" (as checks we mean all steps to add a credential) at this point. We have a clear understanding that the saga logic will eventually include checks in subsequent iterations. In our current development phase, our focus is primarily on setting up the basic structure of the saga and coordinating it with the user interface.

Once we reach the point where we can implementing the "checks" because available from package and it will ensure that it's well-documented and clear in the code.

Reply:
A comment has been added in #5080, thanks.